cmd/hemictl/hemictl.go

```diff
@@ -418,6 +419,9 @@ func tbcdb() error {
 			if mc, err = strconv.ParseUint(maxCache, 10, 64); err != nil {
 				return fmt.Errorf("maxCache: %w", err)
 			}
+			if mc > math.MaxInt32 {
+				return fmt.Errorf("maxcache exceeds max int value")
+			}
 			cfg.MaxCachedTxs = int(mc)
```

Comment:
It appears `strconv#ParseInt` supports providing a `bitSize` of `0`, which will use the platform's `int` bit size (int32/int64) - per https://pkg.go.dev/strconv#ParseInt. This allows us to cast the output to `int` without any issues. This could also be used for all of the other cases where `ParseInt` or `ParseUint` is used and the output is cast to `int` or `uint`.

There also appears to be some cases here where we are using `ParseUint` and casting to an `int`, presumably because we don't want to allow inputs less than zero - In these cases, I think either we should add a bounds check or the config value type should be changed to `uint`.

```suggestion
			if mc, err = strconv.ParseInt(maxCache, 10, 0); err != nil {
				return fmt.Errorf("maxCache: %w", err)
			}
			cfg.MaxCachedTxs = int(mc)
```

Reply:
Ah nice, fixed!